test/compilable/cppmangle.d

```diff
@@ -450,3 +450,20 @@ version (Posix) // all non-Windows machines
     static assert(test37!int.mangleof == "_ZN5SPACE6test37IiEEiv");
 }
 
+/**************************************/
+// 15388
+
+extern (C++) void test15388(typeof(null));
+
+version (linux)
+{
+    static assert(test15388.mangleof == "_Z9test15388Dn");
+}
+version (Win32)
+{
+    static assert(test15388.mangleof == "?test15388@@YAX$$T@Z");
+}
+version (Win64)
+{
+    static assert(test15388.mangleof == "?test15388@@YAX$$T@Z");
+}
```

Comment:
Perhaps on version statement instead, i.e. `Windows`. Or is there a particular reason to have two?

Reply:
I followed the pattern from higher in the file.

Reply:
Ok.

Reply:
I'm into conformity, and It's not the business of this PR to tidy someone else's code.

Reply:
That's absolutely fine.